File: python/find_unique_binary_string.py
# no.1980
class Solution:
    def findDifferentBinaryString(self, nums: List[str]) -> str:
        length = len(nums)
        for i in range(length):
            bin_str = nums[i]
            for j in range(length):
                can = "0" if bin_str[j] == "1" else "1"
                can = bin_str[:j] + can + bin_str[j+1:]
                if can not in nums[i:]:
                    return can

File: go/find_unique_binary_string.go
//no. 1980

func is_in_arr(s string, nums []string) bool{
    for _, i := range nums{
        if i == s{
            return true
        }
    }
    return false
}

func findDifferentBinaryString(nums []string) string {
    length := len(nums)
    var can string
    for i := 0; i < length; i++{
        bin_str := nums[i]
        for j := 0; j < length; j++{
            if bin_str[j] == '0'{
                can = "1"
            }else{
                can = "0"
            }
            can = bin_str[:j] + can + bin_str[j+1:]
            if !is_in_arr(can, nums){
                return can
            }
        }
    }
    return can
}
